pkg/resource/logging-config/alloy/logging-config.alloy.yaml.template

```diff
@@ -58,3 +61,60 @@ alloy:
           path: /run/log/journal
       - name: alloy-tmp
         emptyDir: {}
+podLogs:
+{{- if .IsWorkloadCluster }}
+- name: namespace-default
+  namespace: giantswarm
```

Comment:
```suggestion
  namespace: giantswarm
```

Maybe this should bé in thé kube-system.namespace? Giantswarm namespace will soon disappear on WCs

Reply:
Ok that makes sense, I'll move PodLogs into kube-system namespace

Reply:
> Giantswarm namespace will soon disappear on WCs

Really? I didn't know this :open_mouth: are there some docs about it?

Reply:
Nope but it's coming with the removal of the chart-operator on WCs

Reply:
The goal is to not have any giant swarm code on WCs

Reply:
PodLogs are now created in kube-system namespace